src/main/java/Duke.java

```diff
@@ -1,10 +1,264 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String INPUT_PROMPT = "$ ";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+
+    private static final String TASK_DEADLINE_SPLITTER = "/by";
+    private static final String TASK_EVENT_SPLITTER = "/at";
+
+    private static final String MESSAGE_WALL = "------------------------------------------------------------";
+    private static final String MESSAGE_LIST_EMPTY = "List is empty";
+    private static final String MESSAGE_LIST_HEADER = "Task List:";
+    private static final String MESSAGE_INVALID_TASK_NUMBER = "Invalid task number";
+    private static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
```

Comment:
Good job following the naming convention for constant terms making them all uppercase. This makes it easier to identify them.